lib/crawly/bench.ex

```diff
@@ -55,9 +60,20 @@ defmodule Crawly.Bench do
         } | Current crawl speed is: #{num_requests_now} requests/sec"
       )
 
+      num_of_workers = DynamicSupervisor.count_children(name)[:workers]
+
+      if rem(retries, 10) == 0 and retries != 100 do
+        Logger.info("Adding 10 workers for #{name}")
+
+        Enum.map(1..10, fn _x ->
+          DynamicSupervisor.start_child(name, {Crawly.Worker, [name]})
```

Comment:
I would rather prefer to have it inside crawly. To do something like Crawly.Manager.add_worker/spider_name